### ISSUE TITLE
`this.curDocsView.removeEventListener is not a function` in ResizeObserver in Safari 10.0

### ISSUE THREAD

**Issue by panstromek** [quasarframework/quasar#4870]
**Describe the bug**
When using Quasar app in Safari 10.0, this error shows up in the console. It also happens on Quasar docs page for resize observer.
```
this.curDocsView.removeEventListener is not a function, this.curDocsView.removeEventListener is undefined
```
The offending line is in ResizeObserver in `__cleanup` method, line 59 (in Quasar 1.0.5)
```js
        this.curDocView.removeEventListener('resize', this.trigger, listenOpts.passive)
```

**To Reproduce**
1. Go to https://quasar.dev/vue-components/resize-observer in Safari 10.0

**Platform (please complete the following information):**
OS: Mac OS
Browsers: Safari 10.0

**Comment by rstoenescu** [comment]
Hi,

I currently have Safari 12.1.2 (14607.3.9) and cannot reproduce.
Would you be so kind to investigate a bit on this? This is a weird bug. Can you check what this.curDocView is and what it contains pls?

**Comment by nothingismagick** [comment]
Also track exactly the browser frame height and width.

**Comment by rstoenescu** [comment]
Browser frame height and width is irrelevant.

**Comment by nothingismagick** [comment]
I've seen weird behavior at very specific sizes where the scrollbars appear
and disappear. So in a case where its hard to reproduce I think that any
leads can be helpful.

>

**Comment by nothingismagick** [comment]
Also I noticed in Matyas comment that there is a discrepency:

this.curDocsView.removeEventListener is not a function,
this.curDocsView.removeEventListener is undefined

The offending line is in ResizeObserver in __cleanup method, line 59 (in
Quasar 1.0.5)

        this.curDocView.removeEventListener('resize', this.trigger,
listenOpts.passive)

curDocView and curDocsView

**Comment by panstromek** [comment]
I'll try to investigate a bit more when I get back into that machine again - it's not mine, I just used it to test.

About the discrepancy - It's a typo, I rewrote the error manually, because I didn't have access to any account or smth where I could copypaste in on that MacBook easily ;) if I click on the error it points into that line in ResizeObserver

**Comment by rstoenescu** [comment]
Managed to reproduce this on iOS 10 by chance.
Fix will land in 1.1.0.

Thanks so much for reporting!

**Comment by panstromek** [comment]
Oh, awesome. I haven't had a chance to use any Apple device since reporting this, good ;)
